src/vaadin-grid-column-resizing-mixin.html

```diff
@@ -56,7 +56,11 @@
           var style = window.getComputedStyle(targetCell);
           var minWidth = 10 + parseInt(style.paddingLeft) + parseInt(style.paddingRight) + parseInt(style.borderLeftWidth)
             + parseInt(style.borderRightWidth) + parseInt(style.marginLeft) + parseInt(style.marginRight);
-          column.width = Math.max(minWidth, targetCell.offsetWidth + e.detail.x - targetCell.getBoundingClientRect().right) + 'px';
+          if (this.__isRTL) {
```

Comment:
nit: let's refactor to make this more readable and avoid copy-pasting
```js
const maxWidth = targetCell.offsetWidth + (this.__isRTL ? targetCell.getBoundingClientRect().left - e.detail.x : e.detail.x - targetCell.getBoundingClientRect().right);
column.width = Math.max(minWidth, maxWidth) + 'px';
```

Reply:
Rebased and updated.